src/components/atoms/Title/style.scss

```diff
@@ -0,0 +1,13 @@
+.title {
+    font-family: var(--main-title-font);
+    color: var(--light-gray-color);
+    font-size: 27px;
+    letter-spacing: 3px;
+}
+.title-subtitle {
```

Comment:
It's just css file. Please use sass here and in other scss files as well.

Reply:
changed css files

Reply:
use nesting here